fvtest/sigtest/CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+###############################################################################
+#
+# (c) Copyright IBM Corp. 2016
+#
+#  This program and the accompanying materials are made available
+#  under the terms of the Eclipse Public License v1.0 and
+#  Apache License v2.0 which accompanies this distribution.
+#
+#      The Eclipse Public License is available at
+#      http://www.eclipse.org/legal/epl-v10.html
+#
+#      The Apache License v2.0 is available at
+#      http://www.opensource.org/licenses/apache2.0.php
+#
+# Contributors:
+#    Multiple authors (IBM Corp.) - initial implementation and documentation
+###############################################################################
+
+add_executable(omrsigtest
+	sigTest.cpp
+	sigTestHelpers.cpp
+	main.cpp
+)
+
+#TODO: hack for windows. These libraries should be linked on dependants
+if(OMR_HOST_OS STREQUAL "win")
+	target_link_libraries(omrsigtest
+		ws2_32
+		shell32
+		Iphlpapi
+		psapi
+		pdh
+	)
+endif()
+
+if(OMR_HOST_OS STREQUAL "zos")
+	target_link_libraries(omrsigtest j9a2e)
+endif()
+
+#TODO pretty sure omrtestutil only used for headers
+target_link_libraries(omrsigtest
+	omrGtestGlue
+	omrtestutil
+	j9thrstatic
+	omrport
+	omrglue
+	omrsig
+)
+
+
+#MODULE_INCLUDES += ../util
```

Comment:
broad review comment: please document reason for having all these (and others, where you haven't documented TODO's) blocks of statements commented out.

I tried to identify it where ever I saw it in an attempt to help you, but probably still missed a few spots.